api-reference/10 UI Components/dxPagination/3 Methods/getPageCount().md

```diff
@@ -3,10 +3,10 @@ id: dxPagination.getPageCount()
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Gets the number of all pages.
```

Comment:
Usually, you wouldn't see the word "all" in such descriptions. "Gets the number of elements in the array". 

Not a big deal, but seems redundant. 

```suggestion
Gets the number of pages.
```